scripts/generic-ci-tests.sh

```diff
@@ -99,16 +101,15 @@ case "$TEST_SUITE" in
         ;;
 
     "lms-unit")
-        PAVER_ARGS="--with-flaky --processes=-1 --cov-args='-p' --with-xunitmp"
         case "$SHARD" in
             "all")
-                paver test_system -s lms $PAVER_ARGS
+                paver test_system -s lms $PAVER_ARGS $PARALLEL 2> lms-tests.log
                 ;;
             [1-3])
-                paver test_system -s lms --attr="shard=$SHARD" $PAVER_ARGS
+                paver test_system -s lms --attr="shard=$SHARD" $PAVER_ARGS $PARALLEL 2> lms-tests.$SHARD.log
                 ;;
             4|"noshard")
-                paver test_system -s lms --attr='!shard' $PAVER_ARGS
+                paver test_system -s lms --attr='!shard' $PAVER_ARGS $PARALLEL 2> lms-tests.REST.log
```

Comment:
The name lms-tests.REST.log might be confusing to some people, as the word "REST" could have different meanings.
I vote for naming it lms-tests.4.log, despite it technically being the results for the 4 shard plus tests with unspecified shard numbers.